webworker/webworker-test.js

```diff
@@ -22,8 +25,6 @@ const outputFormatterLengthIs = (assert, length) => (options) => {
   assert.equal(Object.keys(results).length, length);
 };
 
-/* eslint-disable no-undef */
```

Comment:
This change isn't required as part of this, but I was looking at some of the disabled rules and though the hoisted versions were a little more explicit

Reply:
I like it!